### PR TITLE
[11.0][IMP] base_tier_validation: Filter notifications on systray

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
@ageficent @jbeficent

### CODE REVIEW

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

File: base_tier_validation/static/src/js/systray.js [hunk 1]
```diff
@@ -21,12 +19,21 @@ odoo.define('tier_validation.systray', function (require) {
         start: function () {
             this.$reviews_preview = this.$('.o_mail_navbar_dropdown_channels');
             this._updateReviewPreview();
```
Comment (HviorForgeFlow) [inline base_tier_validation/static/src/js/systray.js:21]: @ageficent is this needed?

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@moylop260 could you merge? Thanks!

**Pull request merged**